src/main/java/com/todolist/app/servlet/NewTaskServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.todolist.app.servlet;
+
+
+import com.todolist.app.model.Item;
+import com.todolist.app.model.Priority;
+import com.todolist.app.model.Status;
+import com.todolist.app.model.Tag;
+import com.todolist.app.service.ToDoStoreSingleton;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@WebServlet(name = "NewTaskServlet", value = "/NewTaskServlet")
```

Comment:
As far as I remember the common resource(servlet is a resource) naming convention is to use lowercase letters in addition to dashes to split the words. E.g. https://restfulapi.net/resource-naming/

Also, let's omit words like servlets/controllers in resource paths. In should be `/new-task` not `new-task-servlet` or `new-task-controller`